protocol/communities/manager.go

```diff
@@ -1313,7 +1317,13 @@ func (m *Manager) CheckPermissionToJoin(id []byte, addresses []gethcommon.Addres
 
 	becomeMemberPermissions := community.TokenPermissionsByType(protobuf.CommunityTokenPermission_BECOME_MEMBER)
 
-	hasPermission, err := m.checkPermissionToJoin(becomeMemberPermissions, addresses, false)
+	allChainIDs, err := m.tokenManager.GetAllChainIDs()
+	if err != nil {
+		return nil, err
+	}
+
+	accountsAndChainIDs := combineAddressesAndChainIDs(addresses, allChainIDs)
+	hasPermission, err := m.checkPermissionToJoin(becomeMemberPermissions, accountsAndChainIDs, false)
```

Comment:
@cammellos here you can see the usage of `checkPermissionToJoin()` which now receives an accounts/chainID combination as discussed